packages/strip/README.md

```diff
@@ -36,59 +36,88 @@ export default {
     dir: 'output',
     format: 'cjs'
   },
-  plugins: [
-    strip({
-      labels: ['unittest']
-    })
-  ]
+  plugins: [strip({})]
 };
 ```
 
 Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#command-line-reference) or the [API](https://www.rollupjs.org/guide/en/#javascript-api).
 
+### Default Options
```

Comment:
please remove this section. the defaults are spelled out in the options below

Reply:
I added this because I wished it had been here when I started trying to use this plugin, but ok.